health/micro-ui/web/micro-ui-internals/packages/modules/microplan/src/configs/UICustomizations.js

```
@@ -78,41 +78,63 @@ export const UICustomizations = {
+            const customName = type === "Estimations" ? campaignName : `${campaignName}_DRAFT`;
             Digit.Utils.campaign.downloadExcelWithCustomName({
               fileStoreId: fileId,
-              customName: campaignName
+              customName: customName,
             });
           };
 
-          const onActionSelect = (e) => {
+          const onActionSelect = async (e) => {
             if (e.name === "MP_ACTIONS_EDIT_SETUP") {
               const key = parseInt(row?.additionalDetails?.key);
```

Biome (1.9.4)

[error] Use Number.parseInt instead of the equivalent global.

ES2015 moved some globals into the Number namespace for consistency.
Safe fix: Use Number.parseInt instead.


(lint/style/useNumberNamespace)